.github/workflows/ci.yml

```diff
@@ -19,15 +19,15 @@ jobs:
     name: rustfmt
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
```

Comment:
newbie question, do we need to do so for neon repo as well?

Reply:
Most likely. I'm not entirely sure what it helps but a friend suggested we should update - I think v2 uses deprecated actions features